### ISSUE TITLE
`defaultDate` does not work with a date object

### ISSUE THREAD

**Issue by Eonasdan** [Eonasdan/tempus-dominus#2391]
### Discussed in https://github.com/Eonasdan/tempus-dominus/discussions/2389

<div type='discussions-op-text'>

<sup>Originally posted by **deanstreetlab** September  3, 2021</sup>
According to the docs, there is this [defaultDate](https://getdatepicker.com/6/options.html#defaultDate) option which takes a JS Date arg.  I tried setting it as below but the datepicker stops working. What's wrong in my setting?

```
new tempusDominus.TempusDominus(document.querySelector('#datepicker'), {
    display: {
        viewMode: 'years',
        components: {
            decades: false,
            year: true,
            month: true,
            date: true,
            hours: false,
            minutes: false,
            seconds: false,
         }
   }, //end display options
  restrictions: {
      minDate: new Date(1960, 0, 1),
      maxDate: new Date(2020, 11, 31), 
   }, //end restrictions options
  useCurrent: false,  
  defaultDate: new Date("2019-12-31") //NOT WORKING
 });
```</div>

**Comment by Eonasdan** [comment]
Closing.

**Comment by rczwojdrak** [comment]
Still isn't working

Just check this with Simple Example from examples page:
`new TempusDominus(document.getElementById('datetimepicker1'), {
  defaultDate: new Date("2019-12-31")
});`

**Comment by Eonasdan** [comment]
Hi please check [this stackblitz](https://stackblitz.com/edit/tempus-dominus-v6-22391?file=index.js). If the input is empty and the default date is provided, then the input will be set to the default date.

**Comment by Eonasdan** [comment]
Hello. This should be fixed in 6.2.7

**Comment by khuongletikijob** [comment]
Thanks you
